pkg/apis/core/types_seed.go

```diff
@@ -108,7 +108,20 @@ type SeedBackup struct {
 type SeedDNS struct {
 	// IngressDomain is the domain of the Seed cluster pointing to the ingress controller endpoint. It will be used
 	// to construct ingress URLs for system applications running in Shoot clusters.
+	// If this is set spec.settings.nginxIngress can not be enabled and vise versa.
 	IngressDomain string
+	// NginxIngress controls settings for a gardener managed nginx-ingress controller
+	// +optional
```

Comment:
Remove all `// +optional` comments - they are not needed in the internal API version